fonts.sc

```diff
@@ -0,0 +1,16 @@
+import mill._
+import ammonite.ops._
+
+
+trait FontDownloader extends Module {
+
+  def getFonts = T {
```

Comment:
I think we can restructure the code a bit.
Could you move this function to the `WebpackModule` trait in `webpack.sc`? That's where the web assets and stylesheets are dealt with, so it makes sense to handle fonts there too. I don't think we need a `FontDownloader` trait as it's a single function and we aren't using it in multiple places.

This should return a `PathRef` for where the fonts are downloaded. You can see examples of `PathRefs` in the `WebpackModule`, but I think the following code should do the trick:

```scala
def fonts = T {
  val dest = T.ctx().dest
  val fontsDir = dest / 'fonts
  ...
  PathRef(fontsDir)
}
```

Once you start changing the css to use `@fontface` you'll have a better idea of what this path should be.

I've renamed it from `getFonts` to `fonts` to fit with the existing code, but the name isn't too important.

You can test the download with `mill web.fonts`.